schema/spacewalk/upgrade/susemanager-schema-4.3.3-to-susemanager-schema-4.3.4/003-add-ssh-port-minion.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE suseMinionInfo ADD COLUMN IF NOT EXISTS
+    ssh_push_port NUMERIC;
```

Comment:
As uyuni repo was tagged, the version of susemanager-schema was increased already to 4.3.4.
You need to move this script now into `susemanager-schema-4.3.4-to-susemanager-schema-4.3.5/` directory.

Reply:
Fixed